client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -66,8 +68,24 @@ class ProductCreate extends React.Component {
 	}
 
 	onSave = () => {
-		const { siteId, product } = this.props;
-		this.props.createProduct( siteId, product );
+		const { siteId, product, translate } = this.props;
+
+		const successAction = successNotice(
+			translate( '%(product)s successfully created.', {
+				args: { product: product.name },
+				context: 'eCommerce Product',
+			} ),
+			{ duration: 4000 }
+		);
+
+		const errorAction = errorNotice(
+			translate( 'There was a problem saving %(product)s. Please try again.', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?